src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -444,6 +472,15 @@ def _nfs_add(self, svc_arg, pool, namespace=None):
         orchestrator.raise_if_exception(completion)
         return HandleCommandResult(stdout=completion.result_str())
 
+    @_write_cli('orchestrator rbd-mirror rm',
+                "name=name,type=CephString,req=false",
+                'Remove an MDS service (mds id or fs_name)')
+    def _mds_rm(self, name=None):
```

Comment:
_mds_rm?

Reply:
... and should it be moved up to around line 384 to keep it w/ the other rbd-mirror functions?

Reply:
yeah, i'm not sure why the orchestrator_cli stuff ended up organized the way it did